wireless_modules/middle_module/test.py

```diff
@@ -0,0 +1,64 @@
+import paho.mqtt.client as mqtt
+import time
+import argparse
+
+
+parser = argparse.ArgumentParser(
+    description='MQTT wireless module test script with dummy payloads',
+    add_help=True)
+parser.add_argument(
+    '--module', action='store', type=int,
+    default=2, help="""The module number to determine the publish and subscription topics. 
+    If nothing is selected it will send on module number 2 topics.""")
+parser.add_argument(
+    '--host', action='store', type=str, default="5.196.95.208",
+    help="""Address of the MQTT broker. If nothing is selected it will
+    default to 5.196.95.208 - a test server by mosquito.""")
```

Comment:
locahosts usually always work well, and since this was a test file I wanted to ensure that using an external host works as well. When u say we would use a local broker, do u mean in the end module or just when testing?

Reply:
A local broker is when you host the MQTT broker on your own computer, using something like `mosquitto`. In the final product we will also be using a local broker, hosted on the DAS pi.

Reply:
Oh right, sure I'll make it local host then